spec/helpers/webmock_helpers.rb

```diff
@@ -85,6 +86,24 @@ def contact_find_stub(endpoint, id, success = true)
     end
   end
 
+  def contacts_find_query_stub(endpoint, query, success = true)
+    if success
+      stub_request(
+        :get,
+        "#{endpoint}/private/api/v2/json/contacts/list?query=#{query}")
+        .to_return(
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.